src/plugins/intel_gna/src/frontend/scale_factor_calc.hpp

```diff
@@ -79,21 +69,22 @@ static std::vector<double> getPWLSlopes(const LayerInfo& info) {
  * @return Best output activation scale factor
  */
 static float selectBestOutputScaleFactors(float inScale, std::vector<float> outScales, const std::vector<double>& slopes) {
+    using ov::intel_gna::common::is_fp_eq;
```

Comment:
You can use ov::intel_gna_common for both is_fp_eq and float_to_int.

Reply:
ov::intel_gna is the namespace for all stuff related to GNA plugin.
I would leave it as it is due the fact we usually used convention that subdiretory name is nested namespace name.

Reply:
I meant using using namespace ov::intel_gna::common for both mentioned functions.